### DISCUSSION TITLE
Is there anyway to use twilio auth with  dart?

### DISCUSSION THREAD

**Discussion by growthondev** [supabase/supabase#7012] [Questions] [answered]
Is there anyway to use twilio auth with  dart?
Thank you

**Comment by dshukertjr** [comment, marked as answer]
Hi @growthondev,

So sorry for the late response here. 

Yes, you can configure Twilio on your Supabase project just like how you would do it on Javascript SDK, and you can call the signin method on dart with the phone number like this:

```dart
await supabase.auth.signIn(phone: 'YOUR_PHONE_NUMBER');
```
